backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -570,6 +566,18 @@ export class UserResolver {
     return true
   }
 
+  @Authorized([RIGHTS.QUERY_OPT_IN])
+  @Query(() => Boolean)
+  async queryOptIn(@Arg('optIn') optIn: string): Promise<boolean> {
+    const optInCode = await LoginEmailOptIn.findOneOrFail({ verificationCode: optIn })
+    // Code is only valid for 10minutes
+    const timeElapsed = Date.now() - new Date(optInCode.updatedAt).getTime()
+    if (timeElapsed > parseInt(CONFIG.EMAIL_CODE_VALID_TIME.toString()) * 60 * 1000) {
+      throw new Error(`email already sent less than $(CONFIG.EMAIL_CODE_VALID_TIME} minutes ago`)
```

Comment:
this error message is incorrect?

Reply:
Yes you right, was a copy paste of the other functions!